frontend/src/pages/pipelines/global/experiments/GlobalNoExperiments.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateHeader,
+  EmptyStateFooter,
+  Button,
+} from '@patternfly/react-core';
+import { CubesIcon, PlusCircleIcon } from '@patternfly/react-icons';
+import { ExperimentListTabs } from './const';
+
+type GlobalNoExperimentsProps = {
+  tab: ExperimentListTabs;
+};
+
+const GlobalNoExperiments: React.FC<GlobalNoExperimentsProps> = ({ tab }) => {
+  if (tab === ExperimentListTabs.ARCHIVED) {
+    return (
+      <EmptyState data-testid="global-no-archived-experiments">
+        <EmptyStateHeader
+          titleText="No archived experiments"
+          icon={<EmptyStateIcon icon={CubesIcon} />}
+          headingLevel="h4"
+        />
+        <EmptyStateBody>
+          When you are finished with an experiment, you can archive it in the Active tab. You can
```

Comment:
"Active" is meant to be bold here according to the design.

Reply:
missed that good catch